### DISCUSSION TITLE
小白求助：创建nonebot项目选择使用模板出错

### DISCUSSION THREAD

**Discussion by monink1** [nonebot/discussions#359] [问答]
![image](https://user-images.githubusercontent.com/91892513/228546572-875ea6b8-64ee-4fea-a75b-efbc06fe1a6c.png)

已经尝试过在anaconda当前虚拟环境和cmd里输入netsh winsock reset并重启，但报错依然存在

**Comment by yanyongyu** [comment]
加载适配器列表失败。属于网络问题，你可以尝试换一个网或者挂梯子

**Reply by monink1** [reply]
您好，尝试了挂vpn或者用手机热点或者两个一起用，还是一样报错，是否还有其他解决办法呢？
![image](https://user-images.githubusercontent.com/91892513/228570890-c6dd02a3-6071-4f78-964b-e6894063c72a.png)

**Reply by yanyongyu** [reply]
没有其他办法，这就是网络问题

**Reply by monink1** [reply]
好的，谢谢大佬
